components/discord/sources/new-message/test-event.mjs

```diff
@@ -0,0 +1,50 @@
+/* eslint-disable pipedream/required-properties-description */
+/* eslint-disable pipedream/required-properties-version */
+/* eslint-disable pipedream/required-properties-name */
+/* eslint-disable pipedream/required-properties-key */
+export default {
+  "channelId": "1064400555309802049",
+  "guildId": "1064400554651292156",
+  "id": "1127815108093289054",
+  "createdTimestamp": 1697448563933,
+  "type": "CUSTOM",
+  "system": true,
+  "content": "Hello, Universe",
+  "authorId": "929181930610503735",
+  "pinned": true,
+  "tts": true,
+  "nonce": "1127815106121683419",
+  "embeds": [
+    "1127815107844197211",
+  ],
+  "components": [
+    "component1",
+    "component2",
+  ],
+  "attachments": [
+    "1127815109574708224",
+  ],
+  "stickers": [
+    "sticker1",
+    "sticker2",
+  ],
+  "editedTimestamp": 1697448563933,
+  "webhookId": "1127815103877140481",
+  "groupActivityApplicationId": "1127815101708800256",
+  "applicationId": "1127815099540459031",
+  "activity": "activity1",
+  "flags": 1,
+  "reference": "reference1",
+  "interaction": "interaction1",
+  "cleanContent": "Hello, Universe",
+  "author": "ejohnson-pipedream",
+  "authorID": "929181930610503735",
+  "author_metadata": {
+    "avatar": "avatar.jpg",
+    "bot": true,
+  },
+  "guild": "ejohnson-pipedream's server-3",
```

Comment:
```suggestion
  "guild": "ejohnson-pipedream's server-3",
```

Reply:
Changed "server-3" to "Server-3" to follow title case rule from Google's Developer docs style guide.